src/main/scala/mesosphere/marathon/core/flow/LaunchTokenConfig.scala

```diff
@@ -0,0 +1,16 @@
+package mesosphere.marathon.core.flow
+
+import org.rogach.scallop.ScallopConf
+
+trait LaunchTokenConfig extends ScallopConf {
+  //scalastyle:off magic.number
+
+  lazy val launchTokenRefreshInterval = opt[Long]("launch_token_refresh_interval",
+    descr = "The interval (ms) in which to refresh the launch tokens to --launch_token_count",
+    default = Some(30000))
+
+  lazy val launchTokens = opt[Int]("launch_tokens",
+    descr = "Launch tokens per interval",
+    default = Some(1000))
```

Comment:
Actually, this should not depend that much on cluster size. Or the relationship is more complicated than that. It depends more on how fast the mesos master can handle our updates.

I would say that making this smarter is not essential at this point and if you feel strongly about that I would propose to create a separate issue for that and make it happen soon.